src/04 - Web services.md

```diff
@@ -409,8 +431,18 @@ Content-Type: application/json
   - if you are creating a new object, this must be a temporary ID (e.g. a UUID) that will be mapped to the ID of the created object and returned in the response
 - **version**
   - object version when updating an existing object (may be used for optimistic locking internally)
-- **write** (optional)
+- **fields** (optional)
   - regular object fields (string, date, number, boolean) to write
+- **virksomhetsspesifikkeMetadata**
```

Comment:
Is virksomhetsspesifikkeMetadata optional? If there's no bsm to save, should we pass an empty object {} or omit the field entirely? More importantly, since virksomhetsspesifikkeMetadata is only supported for a few entity types, should it really be a separate field in the transaction object?

Reply:
1. The field is optional and I have marked it accordingly in my subsequent commit.
2. Passing an empty object will delete all business-specific metadata (the equivalent of passing the empty-object alternative for the rest of the field types such as null for boolean, string, and number). Omitting the field will not introduce any changes to the object's business-specific metadata.
3. As agreed, I have moved virksomhetsspesifikkeMetadata to the documentation of *fields*.